tokenpki/cert.go

```diff
@@ -14,7 +14,7 @@ import (
 // SelfSignedRSAKeypair generates a 2048-bit RSA private key and self-signs an
 // X.509 certificate using it. You can set the Common Name in cn and the
 // validity duration with days.
-func SelfSignedRSAKeypair(cn string, days int) (*rsa.PrivateKey, *x509.Certificate, error) {
+func SelfSignedRSAKeypair(cn string, days int64) (*rsa.PrivateKey, *x509.Certificate, error) {
```

Comment:
That's.. a lot of days. :) Curious why the change?

Reply:
I've changed the `days` type to `int64` because it's being converted to a `time.Duration` here:
https://github.com/micromdm/nanodep/blob/077b73dfe75f5cf5ce12601ce1ca59bbb925bb0e/tokenpki/cert.go#L29

And `time.Duration`'s underlying type is `int64`.